flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/JarRequestBody.java

```diff
@@ -114,4 +123,10 @@ public Integer getParallelism() {
     public JobID getJobId() {
         return jobId;
     }
+
+    @Nullable
+    @JsonIgnore
+    public Map<String, String> getFlinkConfiguration() {
+        return flinkConfiguration;
+    }
```

Comment:
Consider returning a non-nullable `Configuration`. This should simplify other operations.